com.github.hugolabe.Wike.json

```diff
@@ -0,0 +1,44 @@
+{
+  "app-id" : "com.github.hugolabe.Wike",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "3.38",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "wike",
+  "finish-args" : [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio"
+  ],
+  "cleanup" : [
+    "/include",
+    "/lib/pkgconfig",
+    "/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
```

Comment:
You probably don't want most of those, as you're not installing any C library that might be installing any of those uneeded files. It should be safe to remove most of them

Reply:
Frankly, I do not master this topic. I have kept what the GNOME Builder template put.
Sorry for my bad (google) english.

Reply:
No worries, feel free to remove all of them :-)

Reply:
Ok. Done. Thanks!